alphagov/paas-cf
# [#115870117] Use FQDN for BOSH

## What

This PR changes the way how bosh-agents are connecting to microBOSH instance. Currently we are using hardcoded IP and we want to use FQDN instead. FQDN is based on `${SYSTEM_DNS_ZONE_NAME}` and points to local IP of BOSH instance. The IP depends on AZ we want to deploy BOSH to, currently AZ is hardcoded to `eu-west-1a` and following story should provide a way of specifying BOSH AZ by environmental variable.  IP to AZ mapping is done by terraform. 

## How to review

- destroy your CF and microBOSH deployment
- run create-bosh-cloudfoundry pipeline
- login to bosh by executing `make dev bosh-cli`
- all VMs should be in `running` state

## Who can review

Not @combor, @keymon, @HenryTK
